netbox/core/views.py

```diff
@@ -642,11 +648,145 @@ def get(self, request):
             response['Content-Disposition'] = 'attachment; filename="netbox.json"'
             return response
 
-        plugins_table = tables.PluginTable(plugins, orderable=False)
+        plugins_table = tables.InstalledPluginTable(plugins, orderable=False)
         plugins_table.configure(request)
 
         return render(request, 'core/system.html', {
             'stats': stats,
             'plugins_table': plugins_table,
             'config': config,
         })
+
+
+#
+# Plugins
+#
+
+def get_local_plugins(plugins):
+    for plugin_name in settings.PLUGINS:
+        plugin = importlib.import_module(plugin_name)
+        plugin_config: PluginConfig = plugin.config
+
+        plugin_module = "{}.{}".format(plugin_config.__module__, plugin_config.__name__)  # type: ignore
+        plugins[plugin_config.name] = {
+            'slug': plugin_config.name,
+            'config_name': None,
+            'name': plugin_config.verbose_name,
+            'tag_line': plugin_config.description,
+            'description_short': plugin_config.description,
+            'author': plugin_config.author or _('Unknown Author'),
+            'created': None,
+            'updated': None,
+            'is_local': True,
+            'is_installed': True,
+            'is_certified': False,
+            'is_community': False,
+            'versions': [],
+        }
+
+    return plugins
+
+
+def get_catalog_plugins(plugins):
+    url = 'https://api.netbox.oss.netboxlabs.com/v1/plugins'
+    session = requests.Session()
+
+    def get_pages():
+        payload = {'page': '1', 'per_page': '25'}
+        first_page = session.get(url, params=payload).json()
+        yield first_page
+        num_pages = first_page['metadata']['pagination']['last_page']
+
+        for page in range(2, num_pages + 1):
+            payload['page'] = page
+            next_page = session.get(url, params=payload).json()
+            yield next_page
+
+    for page in get_pages():
+        for data in page['data']:
+
+            versions = []
+            versions.append(data['release_latest'])
```

Comment:
Don't include this item in the table, as it's a duplicate of an entry in `release_recent_history` and is resulting in multiple appearances in the rendered table.